Python/python.py

```diff
@@ -0,0 +1,39 @@
+import random
+
+def rock_paper_scissors():
+    
+    turn_options = ['Rock', 'Paper', 'Scissors']
+
+    player_turn = input("Your turn: Choose Rock, Paper, or Scissors:") 
```

Comment:
Follow the conventions in the other scripts, this should be:

```suggestion
    player_turn = input("Your Move:") 
```